app/renderer/ui/components/main/sections/wallet/tabs/receive/index.tsx

```diff
@@ -2,13 +2,13 @@ import * as React from "react"
 import Wrapper from "app/renderer/ui/components/wrapper"
 import List from "app/renderer/ui/components/list"
 import { InputDefault } from "app/renderer/ui/components/input"
-import { ButtonDefault } from "app/renderer/ui/components/button"
 import PaymentRequest from "app/renderer/ui/components/paymentRequest"
 import { TabInfo, TabComponent } from "app/renderer/ui/components/main/sections"
 
 import {
   paymentRequest,
 } from "app/renderer/ui/components/main/sections/wallet/MockData"
+import ActionButton from "app/renderer/ui/components/button/action"
```

Comment:
Normally, we are exporting the components in the index of the parent folder, so that the import would like as:
```
import {ActionButton} from "app/renderer/ui/components/button"
```